app/v1/models/response/common/BSASEntries.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.common
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+
+case class BSASEntries(bsasId: String,
+                       requestedDateTime: String,
+                       summaryStatus: String,
+                       adjustedSummary: Boolean)
+
+object BSASEntries {
+
+  implicit val reads: Reads[BSASEntries] = (
+    (JsPath \ "calculationId").read[String] and
+      (JsPath \ "requestedDateTime").read[String] and
+      (JsPath \ "status").read[String] and
```

Comment:
Status is returned from DES as an Enum so you can implement a similar case class to TypeOfBusiness and read this field in as that new type.